src/Pirouette/Term/Syntax/Base.hs

```diff
@@ -371,9 +372,9 @@ type LanguageConstrs lang =
 -- no idea what that is, defining an empty instance will use the default
 -- implementatino of @builtinPrelude = M.empty@, which is a fine definition.
 class (LanguageConstrs lang) => LanguageBuiltins lang where
-  type BuiltinTypes lang :: *
-  type BuiltinTerms lang :: *
-  type Constants lang :: *
+  type BuiltinTypes lang :: Kind.Type
+  type BuiltinTerms lang :: Kind.Type
+  type Constants lang :: Kind.Type
```

Comment:
`Type` is central enough to Haskell that it is customary to import it unqualified. I assume there is an internal type to Pirouette called `Type` as well for you to prefer doing a qualified import, which I would argue is confusing and should probably be changed (but it doesn't need to happen in this PR since it requires more thoughts)

Reply:
Yes exactly, `Data.Kind.Type` conflicts with `Pirouette.Term.Syntax.Base.Type` when imported unqualified.

Reply:
As a compromise, to avoid importing either of `Type`s qualified or renaming the pirouette's `Type`, can we just omit `:: *` here and below? I think ghc will infer the right kind here.

Reply:
Indeed, it's the default value.